ufc/bandit-flags-v1.json

```diff
@@ -1,5 +1,6 @@
 {
     "createdAt": "2024-04-17T19:40:53.716Z",
+    "format": "server",
```

Comment:
could these be SERVER and CLIENT so that they line up with typical Enum conventions?

Reply:
yeahhhhhh, they can 😛